src/init.cpp

```diff
@@ -351,6 +353,8 @@ std::string HelpMessage(HelpMessageMode mode)
 #ifndef WIN32
     strUsage += HelpMessageOpt("-pid=<file>", strprintf(_("Specify pid file (default: %s)"), BITCOIN_PID_FILENAME));
 #endif
+    strUsage += HelpMessageOpt("-prometheusmetrics=<host_name>:<port>", _("Expose node metrics in the Prometheus exposition format. "
+            "An HTTP listener will be started on the configured hostname and port, which responds to GET requests on any request path."));
```

Comment:
```suggestion
            "An HTTP listener will be started on the configured hostname and port, which responds to GET requests on any request path. "
            "SECURITY WARNING: this can potentially compromise privacy; read contrib/metrics/README.md before enabling."));
```